src/features/list-view/styles/MetadataListViewMessage.scss

```diff
@@ -0,0 +1,26 @@
+.metadata-view-header-text-container {
```

Comment:
these class name will also need to be changed once the naming convention is finalized, as they do not currently reflect the current state of the component.